### PR TITLE
Move WS_URL from many src files to global service

### PR DESCRIPTION
**Pull request by oehm-smith**
* Using service that is already in place (done last week)

### CODE REVIEW

File: src/client/app/shared/cors-setup/cors-setup.service.ts [hunk 1]
```diff
@@ -11,22 +12,23 @@ import 'rxjs/add/operator/catch';
 export class CorsSetupService {
   // WS_URL : string = 'http://localhost:8080/geodesy-web-services';
   // WS_URL : string = 'https://dev.geodesy.ga.gov.au'; // dev
-  WS_URL : string = 'https://dev.geodesy.ga.gov.au'; // test
+  // WS_URL : string = 'https://dev.geodesy.ga.gov.au'; // test
```
Comment (lbodor) [inline src/client/app/shared/cors-setup/cors-setup.service.ts]: Remove commented code.

File: src/client/app/shared/cors-site/cors-site.service.ts [hunk 2]
```diff
@@ -12,15 +13,15 @@ import { JsonixService } from '../jsonix/jsonix.service';
 export class CorsSiteService {
   // WS_URL : string = 'http://localhost:8080/geodesy-web-services';
   // WS_URL : string = 'https://dev.geodesy.ga.gov.au'; // dev
-  WS_URL : string = 'https://dev.geodesy.ga.gov.au'; // test
+  // WS_URL : string = 'https://dev.geodesy.ga.gov.au'; // test
```
Comment (lbodor) [inline src/client/app/shared/cors-site/cors-site.service.ts]: Remove comments.

File: src/client/app/shared/jsonix/jsonix.service.ts [hunk 3]
```diff
@@ -43,7 +43,7 @@ let marshaller: any = context.createMarshaller();
 export class JsonixService {
   // WS_URL : string = 'http://localhost:8080/geodesy-web-services';
   // WS_URL : string = 'https://dev.geodesy.ga.gov.au'; // dev
-  static WS_URL: string = 'https://dev.geodesy.ga.gov.au'; // test
+  // static WS_URL: string = 'https://dev.geodesy.ga.gov.au'; // test
```
Comment (lbodor) [inline src/client/app/shared/jsonix/jsonix.service.ts]: Remove comments.

File: src/client/app/shared/site-log/site-log.service.ts [hunk 4]
```diff
@@ -5,14 +5,15 @@ import 'rxjs/add/operator/map';
...
 
 /**
  * This class provides the service with methods to retrieve CORS Setup info from DB.
  */
 @Injectable()
 export class SiteLogService {
   // WS_URL: string = 'http://localhost:8080/geodesy-web-services';
-  WS_URL : string = 'https://dev.geodesy.ga.gov.au'; // dev
+  // WS_URL : string = 'https://dev.geodesy.ga.gov.au'; // dev
   // WS_URL: string = 'https://dev.geodesy.ga.gov.au'; // test
```
Comment (lbodor) [inline src/client/app/shared/site-log/site-log.service.ts]: Remove comments.

**Comment by oehm-smith** [comment]
Done.  Re-review please.

**Pull request merged**